ivy_tests/test_ivy/test_frontends/test_tensorflow/test_general_functions.py

```diff
@@ -1271,3 +1271,57 @@ def test_tensorflow_linspace(
         axis=axis,
         on_device=on_device,
     )
+   
+
+# tile
+@st.composite
+def _multiple_shape_helper(draw):
+    dtypes_n_input, input_catcher, input_dims = draw(
+        helpers.dtype_and_values(
```

Comment:
`dtype_and_values` with `ret_shape=True` returns `input_dtype, input_array, input_shape`
so when you draw `shape` it should have `min_dim_size=len(input_shape), max_dim_size=len(input_shape)`
and then you'd want to do `return (input_dtype, input_array), dt_n_multiples`